lib/solargraph/api_map.rb

```diff
@@ -141,6 +134,23 @@ def self.load directory
       api_map
     end
 
+    # Create an ApiMap with a workspace in the specified directory and cache
+    # any missing gems.
+    #
+    # @param directory [String]
+    # @return [ApiMap]
+    def self.load_with_cache directory
+      api_map = load(directory)
+      return api_map if api_map.uncached_gemspecs.empty?
+
+      api_map.uncached_gemspecs.each do |gemspec|
+        puts "Caching #{gemspec.name} #{gemspec.version}..."
```

Comment:
```suggestion
        $stderr.puts "Caching #{gemspec.name} #{gemspec.version}..."
```

Appreciate this status message!  Maybe we can add one to the RBS ingestion path as well so the user knows where their type info is coming from?

I'm inclined to keep the stdout info as-is for better tool ingestability and leave this sort of status message to stderr - but let me know what you think.  

For instance, I use solargraph with overcommit now (thankfully it looks like it wouldn't break it): https://github.com/apiology/cookiecutter-ruby/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/.git-hooks/pre_commit/solargraph_typecheck.rb

Reply:
Yeah, that was an oversight on my part. I changed it to `Solargraph.logger.info`.